integration/query_test.go

```diff
@@ -1239,87 +1241,125 @@ func TestQueryCommandGetMore(t *testing.T) {
 			require.NotNil(t, tc.firstBatch, "firstBatch must not be nil")
 
 			var findRest bson.D
-			if tc.findBatchSize != nil {
-				findRest = append(findRest, bson.E{Key: "batchSize", Value: tc.findBatchSize})
+			aggregateCursor := bson.D{}
+
+			if tc.firstBatchSize != nil {
+				findRest = append(findRest, bson.E{Key: "batchSize", Value: tc.firstBatchSize})
+				aggregateCursor = bson.D{{"batchSize", tc.firstBatchSize}}
+			}
+
+			aggregateCommand := bson.D{
+				{"aggregate", collection.Name()},
+				{"pipeline", bson.A{}},
+				{"cursor", aggregateCursor},
 			}
 
 			findCommand := append(
 				bson.D{{"find", collection.Name()}},
 				findRest...,
 			)
 
-			var res bson.D
-			err := collection.Database().RunCommand(ctx, findCommand).Decode(&res)
-			require.NoError(t, err)
+			for _, command := range []bson.D{findCommand, aggregateCommand} {
+				var res bson.D
+				err := collection.Database().RunCommand(ctx, command).Decode(&res)
+				require.NoError(t, err)
 
-			v, ok := res.Map()["cursor"]
```

Comment:
A lot of change around here is making linter happy with recent deprecation of Map() https://github.com/FerretDB/FerretDB/actions/runs/5410541556/jobs/9832136007